menpo/shape/pointcloud.py

```diff
@@ -379,11 +379,63 @@ def _from_vector_inplace(self, vector):
         """
         self.points = vector.reshape([-1, self.n_dims])
 
+    def same_space(self, other):
+        r"""
+        Check if two PointClouds have the same shapes
+
+        Parameters
+        ----------
+        other : PointCloud
+            The PointCloud to compare.
+        Returns
+        -------
+        Boolean : True if they have the same shape
+        """
+        return self.points.shape == other.points.shape
+
     def __str__(self):
         return "{}: n_points: {}, n_dims: {}".format(
             type(self).__name__, self.n_points, self.n_dims
         )
 
+    def __add__(self, other):
```

Comment:
As discussed - definitely into a new PR to discuss this API change

Reply:
Ok